packaging/osx/sharedframework/shared-framework-distribution-template.xml

```diff
@@ -26,4 +26,19 @@
     <pkg-ref id="{SharedFxComponentId}.pkg">{SharedFxComponentId}.pkg</pkg-ref>
     <pkg-ref id="{HostFxrComponentId}.pkg">{HostFxrComponentId}.pkg</pkg-ref>
     <pkg-ref id="{SharedHostComponentId}.pkg">{SharedHostComponentId}.pkg</pkg-ref>
+    <installation-check script="openSSLCheck()" />
+    <script><![CDATA[
+        function openSSLCheck()
+        {
+            var openSSLRoot = "/usr/local/opt/openssl/lib/";
```

Comment:
This does seem more user-friendly.  But since it makes life hard for macports and non-standard-path-homebrew users I'd personally cut it over making it be too restrictive.

If can't you read an environment variable into openSSLRoot, then do

```
my.result.message = "OpenSSL was not detected at '" + openSSLRoot + "'. Please install OpenSSL to continue. (etc)";
```
?